### ISSUE TITLE
Join Planet Ethereum

### ISSUE THREAD

**Issue by s1na** [ethberlin-hackathon/ETHBerlin-Teambuilding#30]
**TL;DR:** We're starting a community dedicated to the Ethereum ecosystem; we have some ideas, but are also open to new ideas; No matter your background, if you're excited about our goals, we'll be glad to have you onboard :)

#### Greetings
Hey everyone! :)
[Yotam](https://github.com/hemulin) and me are super excited to be participating in ETHBerlin, to get to know you all, and to buidl!

#### About Us
We ♥ Ethereum, and would like to contribute in our own small way to the flourishing ecosystem, by focusing on devtools/infrastructure, education and more.

We also hope that Ethereum will have a **positive effect** on everyone, especially that it increases equal opportunities, and are going to try our best to promote these values.

Being new to the ecosystem ourselves, we know the challenges of wrapping one's head around these difficult concepts, and the joy of euphoria when a solution is found. By coming **together**  and **sharing**  our experiences, we could all hopefully have more euphoria and less frustration. 

You can read our vision, mission and values in the [community](https://github.com/planet-ethereum/community) page.

#### About You
If you're also excited about our vision, and want to put your skills, _whatever_ they may be, to achieving it, you're more than welcome to join us :) If you're not sure how you might help, [write](https://gitter.im/planet-ethereum/Lobby) us, and we'll find a way together ;)

#### Ideas
We have a couple of [ideas](https://github.com/planet-ethereum/community/issues), and we're already working on some [others](https://github.com/planet-ethereum). In particular, we're quite excited about enabling **event-driven smart contract development** at the moment (read the [proposal](https://github.com/planet-ethereum/call-me/wiki/About)).

But, we're quite open to new ideas. That's why, we'd rather wait until the team is complete, and based on everyone's interest choose the idea that we'll be doing at the hackathon. If you think you have an idea that fits our profile, don't hesitate to contact us.

#### Communication
You can find us in the `ETHBerlin - Hackers` Riot channel as @hemulin and @sinamahmoodi, or you can drop by our [gitter](https://gitter.im/planet-ethereum/Lobby). Or write a comment here.

**Comment by 0xGabi** [comment]
Hi my name is Gabriel I really like the ethos of the team and the idea you are trying to implement for the hackathon. I been learning about ethereum for about a year and I just finished the ConsenSys Developer Program. I have basic knowledge of front-end and had the pleasure to attend ETHWaterloo and ETHBuenosAires. I will talk you on gitter also

**Comment by s1na** [comment]
@galactusss Hey Gabriel. Thanks for writing, it's always nice to meet like-minded people. Sure, let's talk on gitter then.
